usaspending_api/etl/elasticsearch_loader_helpers/controller.py

```diff
@@ -99,7 +97,7 @@ def complete_process(self) -> None:
 
         close_all_django_db_conns()
 
-        if self.config["is_incremental_load"]:
+        if self.config["is_incremental_load"] and not self.config["deletes_only"]:
```

Comment:
Isn't `self.config["is_incremental_load"]` False if --deletes-only was provided per LN 178 in usaspending_api/etl/management/commands/elasticsearch_indexer.py?

Reply:
You're right. I believe I added this before I decided to update the `parse_cli_args` method.